Application/EdFi.Ods.WebApi/Program.cs

```diff
@@ -19,8 +19,16 @@ public class Program
     {
         public static async Task Main(string[] args)
         {
-            AssemblyLoaderHelper.LoadAssembliesFromExecutingFolder();
-
+            using (var config = new ConfigurationBuilder()
+                       .AddJsonFile("appsettings.json", true)
+                       .AddJsonFile("appsettings.Development.json", true)
```

Comment:
Technically, we should check the environment before adding the _development_ settings to this configuration. I did a little poking and I'm thinking the approach below would be cleaner and also leverage out-of-the-box behavior for incorporating other default configuration sources appropriately (such command-line arguments and environment variables):
```csharp
using Microsoft.Extensions.DependencyInjection;
...
using (var hostBuilderInitial = Host.CreateDefaultBuilder(args).Build())
{
    var config = hostBuilderInitial.Services.GetService<IConfiguration>();
    AssemblyLoaderHelper.LoadAssembliesFromFolder(
        AssemblyLoaderHelper.GetPluginFolder(
            config?.GetValue<string>("Plugin:Folder") ?? string.Empty));
}
```